packages/dd-trace/src/plugins/plugin.js

```diff
@@ -70,7 +70,9 @@ module.exports = class Plugin {
 
   // TODO: Implement filters on resource name for all plugins.
   /** Prevents creation of spans here and for all async descendants. */
-  skip () {
+
+  skip (ctx = null) {
+    if (ctx) return ctx
```

Comment:
```suggestion
  skip () {
```

As follow-up of the comment above.

Reply:
I removed the `this.skip()` call, I don't believe we need it. @rochdev Can you confirm whether we need the `this.skip` call within the redis plugin? It simply does `storage('legacy').enterWith({ noop: true })`, which was preventing the later redis calls from being traced despite not being on the blocklist.

Reply:
okay, makes sense, i made the change to return `{ noop: true }`